zppy/__main__.py

```diff
@@ -167,8 +168,12 @@ def _validate_config(config):
     validator = Validator()
 
     result = config.validate(validator)
+    print(type(result))
```

Comment:
`result` is a `bool` now. In the past, it has historically been some sort of dictionary or dictionary-like object. For example, from a validation failure discussed in a 3/22 Slack message, we had:
```
Validation results={'e3sm_diags': True, 'e3sm_diags_vs_model': True, 'mpas_analysis': True, 'default': True, 'climo': {'qos': True, 'nodes': True, 'walltime': True, 'input_files': True, 'fre
quency': True, 'mapping_file': True, 'grid': True, 'years': False, 'exclude': True, 'vars': True, 'atm_monthly_180x360_aave': True}, 'ts': True, 'tc_analysis': True, 'amwg': True, 'global_ti
me_series': {'input_subdir': True, 'qos': True, 'nodes': True, 'walltime': True, 'color': True, 'years': True, 'ts_num_years': True, 'figstr': True, 'moc_file': True, 'experiment_name': True
, 'ts_years': False, 'climo_years': False, 'atmosphere_only': True}}
```